### PR TITLE
Hotkeys for project settings

### PR DESCRIPTION
**Pull request by blairanderson**
I added tooltips and hotkeys to bring up the settings for each. 

if you type:
    fn + control + option + 1

then the options should show up

### CODE REVIEW

**Comment by blairanderson** [comment]
WAIT. PLEASE DO NOT MERGE. i'm still working on this

**Comment by blairanderson** [comment]
:+1:

**Comment by blairanderson** [comment]
:+1:  yup this works and its legit.

**Comment by blairanderson** [comment]
ok this is not working yet :-1:

File: spec/support/object_creation_methods.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def new_user(overrides = {})
     defaults = {
       full_name: "Blair Anderson",
       login: 'blairand',
-      token: 'wheeeeee'
```
Comment (gschorkopf) [inline spec/support/object_creation_methods.rb:29]: Awwww

**Pull request merged**